packages/core/contracts/polygon/OracleChildTunnel.sol

```diff
@@ -109,13 +110,18 @@ contract OracleChildTunnel is OracleBaseTunnel, OracleAncillaryInterface, FxBase
      * @param requester sender of the initial price request.
      * @return the stamped ancillary bytes.
      */
-    function stampAncillaryData(bytes memory ancillaryData, address requester) public view returns (bytes memory) {
+    function stampAncillaryData(bytes memory ancillaryData, address requester)
+        public
+        view
+        nonReentrantView()
+        returns (bytes memory)
+    {
         return _stampAncillaryData(ancillaryData, requester);
     }
 
     /**
-     * @dev We don't handle specifically the case where `ancillaryData` is not already readily translateable in utf8.
-     * For those cases, we assume that the client will be able to strip out the utf8-translateable part of the
+     * @dev We don't handle specifically the case where `ancillaryData` is not already readily translatable in utf8.
```

Comment:
- In `OracleChildTunnel` on [line 117](https://github.com/UMAprotocol/protocol/blob/d146d4feddcf99e157a909032a3071e95ca51eb9/packages/core/contracts/polygon/OracleChildTunnel.sol#L117) and on [line 118](https://github.com/UMAprotocol/protocol/blob/d146d4feddcf99e157a909032a3071e95ca51eb9/packages/core/contracts/polygon/OracleChildTunnel.sol#L118) "translateable" should be "translatable".